values.yaml

```diff
@@ -112,6 +112,12 @@ runner:
       # tag: "latest"
       # pullPolicy: Always
 
+    # List of namespaces that waypoint can access and deploy into.
+    # By default, the waypoint on-demand runner will create a rolebinding granting the
+    # edit clusterrole in the namespace the chart is installed into. Waypoint will
+    # instead create this rolebinding in each managed namespace.
```

Comment:
Should this be clarified that if the user wants to also manage the namespace Waypoint is installed into, they must explicitly add it if this is set? It wasn't immediately clear to me whether it was implicit or not.